### ISSUE TITLE
Fails to deserialize Option<Vec<i32>> if only contains a single element

### ISSUE THREAD

**Issue by dvdmgl** [jplatte/serde_html_form#19]
with the code:
```rust
    #[derive(Deserialize, PartialEq, Debug)]
    struct Form {
        value: Option<Vec<i32>>,
    }

    assert_eq!(serde_html_form::from_str(""), Ok(Form { value: None }));
    assert_eq!(serde_html_form::from_str("value=1&value=2"), Ok(Form { value: Some(vec![1, 2]) }));
    assert_eq!(serde_html_form::from_str("value=1"), Ok(Form { value: Some(vec![1]) })); // TEST FAILS
```
returns `Err(Error("invalid type: string \"1\", expected i32"))`

**Comment by jplatte** [comment]
Which version of `serde_html_form` is this?

**Comment by jplatte** [comment]
Fixed in 2aca38427893b7051f5bfb9e73248e23937f45fb, fix released in v0.2.6.
